app/src/main/java/org/simple/clinic/analytics/HeapAnalyticsReporter.kt

```diff
@@ -3,18 +3,20 @@ package org.simple.clinic.analytics
 import android.content.Context
 import com.heapanalytics.android.Heap
 import org.simple.clinic.BuildConfig
+import org.simple.clinic.user.User
 
 class HeapAnalyticsReporter(context: Context, debug: Boolean = false) : AnalyticsReporter {
 
   init {
     Heap.init(context.applicationContext, BuildConfig.HEAP_ID, debug)
   }
 
-  override fun setUserIdentity(id: String) {
-    Heap.identify(id)
+  override fun setLoggedInUser(user: User, isANewRegistration: Boolean) {
```

Comment:
`isANewRegistration` seems unused?

Reply:
Weird, something seems to gone wrong when I rebased on `master`. Will check.

Reply:
Oh wait, nothing has gone wrong.

`isANewRegistration` is used only in `MixpanelAnalyticsReporter` because Heap doesn't need anything different to be done when a user signs up vs logs in. `Mixpanel` needs an additional call to be made for a sign up so this parameter is being used there.

Reply:
I did consider having two separate methods on the reporters themselves, but I decided not to since the `Analytics` does have two different methods for this use case, and no class will use the reporters directly. They will always call the `Analytics` methods themselves.